contracts/Gate.sol

```diff
@@ -259,18 +260,18 @@ contract Gate is ProtocolVersioned, Owned {
       //Transfer bounty amount to the msg.sender account
       require(OpenSTValueInterface(openSTProtocol).valueToken().transfer(msg.sender, bounty));
     }
-
+      uint256 stakeProcessedAmount = stakeRequest.amount;
     // delete the stake request from the mapping storage
     delete stakeRequests[staker];
 
-    return true;
+    return stakeProcessedAmount;
   }
 
 
   function revertStaking(
     bytes32 _stakingIntentHash)
     external
-    returns (bool /* success */)
+    returns (uint256 /* amount */)
```

Comment:
similar comment; name of variable should remain `stakeRequestAmount` to be consistent with other two functions